### PR TITLE
Fix deployment

### PR DESCRIPTION
**Pull request by shyeavr**
finish percent calculation redesign and elevation test

### CODE REVIEW

File: tests/pycsw/test_data/queries.py
Comment (dannysigalovich) [inline tests/pycsw/test_data/queries.py:1]: Why you removed new line marker?

File: tests/proactive/csw_wmts-proactive.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import os
+import random
+import sys
+from pathlib import Path
+
+from common.data_modules.random_tiles_generator import create_random_layers_urls
+from tests.pycsw.test_data.queries import POLYGON_XML, ID_RECORD_XML, REGION_RECORD_XML
+
+myDir = os.getcwd()
+sys.path.insert(0, '../..')
```
Comment (dannysigalovich) [inline tests/proactive/csw_wmts-proactive.py:10]: setup.py won't work ?

**Review by dannysigalovich — CHANGES_REQUESTED** [review]
Look at the comments i have added. and i forgot to add comment of the variables in camel case. 
python should use underscore. myDir => my_directory / my_dir

**Review by dannysigalovich — CHANGES_REQUESTED** [review]
Approve the changes i requested

File: tests/proactive/csw_wmts-proactive.py (hunk 1 above)
Comment (shyeavr) [reply tests/proactive/csw_wmts-proactive.py:10]: what?

**Review by dannysigalovich — APPROVED** [review]

**Pull request merged**